docs/resources/User.md

```diff
@@ -181,7 +181,7 @@ Modify the requester's user account settings. Returns a [user](#DOCS_RESOURCES_U
 
 ## Get Current User Guilds % GET /users/@me/guilds
 
-Returns a list of partial [guild](#DOCS_RESOURCES_GUILD/guild-object) objects the current user is a member of. Requires the `guilds` OAuth2 scope.
+Returns a list of partial [guild](#DOCS_RESOURCES_GUILD/guild-object) objects the current user is a member of. For OAuth2, requires the `guilds` scope.
```

Comment:
What is this change trying to indicate?

Reply:
It's to indicate that this endpoint isn't restricted to OAuth2, you can call it with a bot token too.
See the previous review: https://github.com/discord/discord-api-docs/pull/6356#discussion_r1294407257

Reply:
Yeah that ^

Reply:
See also https://discord.com/channels/@me/1100097561135685725/1140820115647103158

Reply:
ahh i see, makes sense ~